apis/ec2/v1beta1/securitygroup_types.go

```diff
@@ -24,13 +24,9 @@ import (
 // SecurityGroupParameters define the desired state of an AWS VPC Security
 // Group.
 type SecurityGroupParameters struct {
-	// TODO(muvaf): Region is a required field but in order to keep backward compatibility
-	// with old Provider type and not bear the cost of bumping to v1beta2, we're
-	// keeping it optional for now. Reconsider before v1beta2 or v1.
 
 	// Region is the region you'd like your SecurityGroup to be created in.
-	// +optional
-	Region *string `json:"region,omitempty"`
+	Region *string `json:"region"`
```

Comment:
Required fields should values, not pointers.

```suggestion
	Region string `json:"region"`
```